src/txn/txn.c

```diff
@@ -1627,22 +1627,34 @@ __wt_verbose_dump_txn_one(WT_SESSION_IMPL *session, WT_TXN *txn)
 		break;
 	}
 	WT_RET(__wt_msg(session,
-	    "mod count: %u"
+	    "transaction id: %" PRIu64
+	    ", mod count: %u"
 	    ", snap min: %" PRIu64
 	    ", snap max: %" PRIu64
+	    ", no of named snapshots: %u"
```

Comment:
This is not quite correct. I suggest just calling it `snapshot count`.

Reply:
Corrected.